source/index.md

```diff
@@ -1,16 +1,23 @@
 # <i class="fas fa-home"></i> Before you start
 
-[OpenFisca](https://openfisca.org) is an open source engine to write rules as code.
+## What is OpenFisca
 
-Describe your tax and benefit system, provide a situation as input (i.e income), ask for a calculation as output (i.e. income tax), and get your results.
+[OpenFisca](https://openfisca.org) is an open-source engine to write rules as code.
+
+Describe your tax and benefit system, provide a situation as input (i.e income), ask for a calculation as output (i.e. income tax), and get your results!
 
 ## Who uses OpenFisca
 
-* **Economists and lawmakers**: calculate the effects of policies. Combine them with survey data to simulate the impact of a reform on a given government’s budget and on a population’s standard of living.
-* **Developers and companies**: easily create web applications based on simulation results, thanks to the [web API](openfisca-web-api/index.md). You can build a great variety of services by coding formulas, hosting your own instance and building your own extensions.
-* **Public administrations**: stop building your own micro-simulation software and tax & benefit calculators. Instead, contribute to OpenFisca, collaborate with other administrations and reduce costs to the taxpayer.
+* **Economists and lawmakers**:
+Economists and lawmakers make use of OpenFisca to calculate the effects of policies. By combining them with survey data to simulate the impact of reform on a given government’s budget and a population’s standard of living.
+
+* **Developers and companies**:
+Developers and companies use Openfisca to effortlessly create web applications based on simulation results with the help of the OpenFisca [web API](openfisca-web-api/index.md). You can build a great significant variety of services by coding formulas, hosting your instances and building your extensions.
 
-## Path to using OpenFisca
+* **Public administrations**:
+Stop building your micro-simulation software and tax & benefit calculators from scratch. Join OpenFisca, contribute to OpenFisca, collaborate with other administrations and reduce costs paid by the taxpayers.
+
+## Ways to use OpenFisca
```

Comment:
The three listed points are sequential and not alternatives 🙂 

```suggestion
## Way to use OpenFisca
```

Reply:
okay Got it